stdlib/contrib/bonitoo-io/tickscript/tickscript.flux

```diff
@@ -0,0 +1,95 @@
+package tickscript
+
+import "experimental"
+import "influxdata/influxdb"
+import "influxdata/influxdb/monitor"
+import "influxdata/influxdb/schema"
+import "universe"
+
+// alert
+alert = (
+    check,
+    id=(r)=>"${r._check_id}",
+    message=(r)=>"Check: ${r._check_name} is: ${r._level}",
+    details=(r)=>"",
+    crit=(r) => false,
+    warn=(r) => false,
+    info=(r) => false,
+    ok=(r) => true,
+    tables=<-) =>
+  tables
+    |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
+    |> map(fn: (r) => ({ r with id: id(r: r) }))
+    |> map(fn: (r) => ({ r with details: details(r: r) }))
+    |> monitor.check(
+        crit: crit,
+        warn: warn,
+        info: info,
+        ok: ok,
+        messageFn: message,
+        data: check
+    )
```

Comment:
```
// alert
alert = (
    check,
    id=(r)=>"${r._check_id}",
    message=(r)=>"Check: ${r._check_name} is: ${r._level}",
    details=(r)=>"",
    crit=(r) => false,
    warn=(r) => false,
    info=(r) => false,
    ok=(r) => true,
    topic="",
    tables=<-) => {
       data = if topic != "" then
           (tables=<-) => tables 
                |> set(key: "_topic", value: name )
                |> experimental.group(mode: "extend", columns: ["_topic"])
           else 
               (tables=<-) => tables 
      return tables
        |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
        |> map(fn: (r) => ({ r with id: id(r: r) }))
        |> map(fn: (r) => ({ r with details: details(r: r) }))
        |> addTopic()
        |> monitor.check(
            crit: crit,
            warn: warn,
            info: info,
            ok: ok,
            messageFn: message,
            data: check
        )
    }
    ```

Reply:
also this might work:

```
// alert
alert = (
    check,
    id=(r)=>"${r._check_id}",
    message=(r)=>"Check: ${r._check_name} is: ${r._level}",
    details=(r)=>"",
    crit=(r) => false,
    warn=(r) => false,
    info=(r) => false,
    ok=(r) => true,
    topic="",
    tables=<-) => {
       data = if topic != "" then
               {check with tags: {check.tags with _topic: topic}
           else 
               check 
      return tables
        |> drop(fn: (column) => column =~ /_start.*/ or column =~ /_stop.*/)
        |> map(fn: (r) => ({ r with id: id(r: r) }))
        |> map(fn: (r) => ({ r with details: details(r: r) }))
        |> monitor.check(
            crit: crit,
            warn: warn,
            info: info,
            ok: ok,
            messageFn: message,
            data: data,
        )
    }
    ```